plancheck/expect_non_empty_plan_test.go

```diff
@@ -9,26 +9,83 @@ import (
 
 	r "github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func Test_ExpectNonEmptyPlan(t *testing.T) {
+func Test_ExpectNonEmptyPlan_OutputChanges_None(t *testing.T) {
```

Comment:
Nit: Should `_None` be used here? Seems like it's indicating that there are no output changes?

Reply:
Good shout -- updated the test naming.